### PR TITLE
Reformatted instructions for the class ES6 challenges

### PR DESCRIPTION
**Pull request by IanTeo**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no):

Progress towards issue #12698, which is a compiled list of QA tasks for the new ES6 section introduced in beta

#### Description
* Added \<blockquote\> tag for multi-line code examples
* Added \<dfn\> and \<code\> tags for important terms
* Replaced "Instructions" with \<hr\> tag
* Capitalised first letters of the SpaceShuttle class
* Added missing semicolon for class Syntax
* Did not make any changes to the tests

### CODE REVIEW

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 1]
```diff
@@ -673,30 +673,22 @@
...
-        "<code>  }</code>",
-        "<code>}</code>",
-        "<code>const zeus = new spaceShuttle('Jupiter');</code>",
-        "Notice that the class keyword declares a new function, and a constructor was added, which would be invoked when new is called - to create a new object.",
-        "Instructions",
-        "Use class keyword and write a proper constructor to create the vegetable class.",
+        "<blockquote>class SpaceShuttle {<br>  constructor(targetPlanet){<br>    this.targetPlanet = targetPlanet;<br>  }<br>}<br>const zeus = new spaceShuttle('Jupiter');</blockquote>",
+        "Notice that the <code>class</code> keyword declares a new function, and a constructor was added, which would be invoked when <code>new</code> is called - to create a new object.",
+        "<hr>",
+        "Use <code>class</code> keyword and write a proper constructor to create the Vegetable class.",
```
Comment (Greenheart) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]:
```diff
-Vegetable
+<code>Vegetable</code>
```

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 2]
```diff
@@ -673,30 +673,22 @@
...
         "The Vegetable lets you create a vegetable object, with a property name, to be passed to constructor."
```
Comment (Greenheart) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: 1. 
```diff
-Vegetable
+<code>Vegetable</code>
```

2. 
```diff
-name
+<code>name</code>
```

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 3]
```diff
@@ -714,30 +706,14 @@
       "title": "class getters and setters",
       "description": [
         "You can obtain values from an object, and set a value of a property within an object.",
-        "These are classically called getters and setters.",
+        "These are classically called <dfn>getters and setters</dfn>.",
```
Comment (Greenheart) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]:
```diff
-<dfn>getters and setters</dfn>
+<dfn>getters</dfn> and <dfn>setters</dfn>
```

**Review by Greenheart — CHANGES_REQUESTED** [review]
@IanTeo Great work! :blush: :+1:

**Comment by IanTeo** [comment]
I have made the requested changes

File: seed/challenges/02-javascript-algorithms-and-data-structures/es6.json [hunk 4]
```diff
@@ -714,31 +706,15 @@
...
-        "<code>console.log(lol.writer);</code>",
+        "<blockquote>class Book {<br>  constructor(author) {<br>    this._author = author;<br>  }<br>  // getter<br>  get writer(){<br>    return this._author;<br>  }<br>  // setter<br>  set writer(updatedAuthor){<br>    this._author = updatedAuthor;<br>  }<br>}<br>const lol = new Book('anonymous');<br>console.log(lol.writer);<br>lol.writer = 'wut';<br>console.log(lol.writer);</blockquote>",
         "Notice the syntax we are using to invoke the getter and setter - as if they are not even functions.",
         "Getters and setters are important, because they hide internal implementation details.",
-        "Instructions",
-        "Use class keyword to create a Thermostat class. The constructor accepts Farenheit temperature.",
-        "Now create getter and setter in the class, to obtain the temperature in Celsius scale.",
+        "<hr>",
+        "Use <code>class</code> keyword to create a Thermostat class. The constructor accepts Farenheit temperature.",
+        "Now create <dfn>getter and setter</dfn> in the class, to obtain the temperature in Celsius scale.",
```
Comment (erictleung) [inline seed/challenges/02-javascript-algorithms-and-data-structures/es6.json]: [Same change suggestion](https://github.com/freeCodeCamp/freeCodeCamp/pull/13559/files#r103077540) should be made here as well.

```diff
- <dfn>getter and setter</dfn>
+ <dfn>getter</dfn> and <dfn>setter</dfn>
```

**Review by erictleung — CHANGES_REQUESTED** [review]

**Comment by IanTeo** [comment]
Sorry, I missed that. I have made the changes requested

**Review by Greenheart — APPROVED** [review]
@IanTeo Thanks for fixing this! I've tested this locally and it looks good! :smile: :+1:

**Pull request merged**